fe/src/test/java/org/apache/doris/planner/QueryPlanTest.java

```diff
@@ -725,4 +725,58 @@ public void testJoinPredicateTransitivityWithSubqueryInWhereClause() throws Exce
         Assert.assertTrue(explainString.contains("CROSS JOIN"));
         Assert.assertTrue(!explainString.contains("PREDICATES"));
     }
+
+    @Test
+    public void testConstantInPredicate() throws Exception {
```

Comment:
please move the test to `ConstantExpressTest`

Reply:
OK